lib/osut/utils.rb

```diff
@@ -4100,48 +4235,62 @@ def getRealignedFace(pts = nil)
     pts = poly(pts, false, true)
     return out if pts.empty?
     return invalid("aligned plane", mth, 1, DBG, out) unless xyz?(pts, :z)
+    return invalid("clockwise pts", mth, 1, DBG, out)     if clockwise?(pts)
+
+    o   = OpenStudio::Point3d.new(0, 0, 0)
+    w   = width(pts)
+    h   = height(pts)
+    d   = h > w ? h : w
+    sgs = {}
+    box = boundedBox(pts)
```

Comment:
The method rests on _boundedBox_, as a means to re-_align_ a flat (or pre-flattened) surface (e.g. via _OpenStudio::Transformation.alignFace_). The goal is to go from an _aligned_ surface (SEB roof):
<img width="1014" alt="seb_aligned" src="https://github.com/rd2/osut/assets/98104985/2206116b-0401-4712-bbdc-b777f2a1242e">
...which would yield the following skylight insertion:
<img width="1311" alt="sloped" src="https://github.com/rd2/osut/assets/98104985/5d43427b-abcf-44c0-8ddc-7a3945f495c7">
... to a more sound, intuitive alignment along its _bounded box_:
<img width="871" alt="OS_skyarray" src="https://github.com/rd2/osut/assets/98104985/93560134-e078-4067-946b-dfa3d7c698b6">